src/Sulu/Component/Webspace/Settings/SettingsManager.php

```diff
@@ -41,9 +49,13 @@ public function save($webspaceKey, $key, $data)
             $value = json_encode($data);
         }
 
-        $this->sessionManager->getWebspaceNode($webspaceKey)->setProperty($propertyName, $value);
+        $this->sessionManager->setNodeProperty(
+            $this->deprecatedSessionManager->getWebspacePath($webspaceKey),
```

Comment:
Actually these calls should be considered deprecated, there is an issue for that: https://github.com/sulu/sulu/issues/2772